src/PicOfTheDay.vue

```diff
@@ -1,36 +1,37 @@
 <template> 
   <section class='potd-component'>
     <iframe
-      :src="url" 
+      :src="potd.url" 
       class='video' 
-      v-if="media_type === 'video'" 
-      :title="title"
+      v-if="potd.media_type === 'video'" 
+      :title="potd.title"
       allowfullscreen="allowfullscreen"
       mozallowfullscreen="mozallowfullscreen" 
       msallowfullscreen="msallowfullscreen" 
       oallowfullscreen="oallowfullscreen" 
       webkitallowfullscreen="webkitallowfullscreen"
     />
-    <img :src="url" class='photo' v-if="media_type === 'image'" :alt="title">
-    <h2>{{ date }}</h2>
-    <h3>{{ title }}</h3>
-    <h4>{{ explanation }}</h4>
+    <img :src="potd.url" class='photo' v-if="potd.media_type === 'image'" :alt="potd.title">
+    <h2>{{ potd.date }}</h2>
+    <h3>{{ potd.title }}</h3>
+    <h4>{{ potd.explanation }}</h4>
```

Comment:
Is this how these work? I thought the v-bind: was used on data properties, like :src would be a data property. Here we only have potd as a property so is that binding at all inside of the template?

Reply:
If you're assigning dynamic values to common html attributes, you have to bind them by using `v-bind:` or `:` for short